cellxgene_schema_cli/cellxgene_schema/cli.py

```diff
@@ -34,7 +34,8 @@ def schema_cli():
     type=click.Path(exists=False, dir_okay=False, writable=True),
 )
 def schema_validate(h5ad_file, add_labels_file):
-    if validate.validate(h5ad_file, add_labels_file):
+    is_valid, errors = validate(h5ad_file, add_labels_file)
```

Comment:
nit: 
```suggestion
    is_valid, _ = validate(h5ad_file, add_labels_file)
```

Reply:
Though, the code for printing errors to stdout might be might better moved here, since it's CLI-specific. That would leave the underlying validate() call to just generate and maintain the errors for use by the caller.  Now that we have a non-CLI usage, that seems reasonable. Pretty minor!

Reply:
We might want to run that past @pablo-gar I'm not sure if he prefers/expects the errors to print in a particular order

Reply:
Doesn't it help to print the errors even if not called from the CLI? Specially now that during upload the CLI is not used.